libs/velo-external-db-core/test/drivers/auth_middleware_test_support.ts

```diff
@@ -1,9 +1,36 @@
+import { WixDataFacade } from '../../src/web/wix_data_facade'
+import * as jwt from 'jsonwebtoken'
+import { decodeBase64 } from '../../src/utils/base64_utils'
+import { authConfig } from '@wix-velo/test-commons'
 
-export const requestBodyWith = (secretKey: string, role?: string | undefined, path?: string | undefined) => ({
+
+export const requestBodyWith = (role?: string | undefined, path?: string | undefined, authHeader?: string | undefined) => ({
     path: path || '/',
     body: {
         requestContext: {
             role: role || 'OWNER',
             settings: {
-                secretKey: secretKey
-            } } } } )
+            } } },
+    header(_name: string) { return authHeader }
+} )
+
+export const signedToken = (payload: Object, expiration: string = '10000ms') =>
```

Comment:
it's not critical, but once you gave a default value to a variable, there's no need to add its type